metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -1175,11 +1091,9 @@ def loadManifestAndCatalog(
         else:
             sources_results = {}
 
-        manifest_schema = dbt_manifest_json.get("metadata", {}).get(
-            "dbt_schema_version"
-        )
-        manifest_version = dbt_manifest_json.get("metadata", {}).get("dbt_version")
-        manifest_adapter = dbt_manifest_json.get("metadata", {}).get("adapter_type")
+        manifest_schema = dbt_manifest_json["metadata"].get("dbt_schema_version")
```

Comment:
Will this throw a keyError (will `metadata` key be always present)?

Reply:
yes it should always be present - I checked the dbt json schemas